realsense2_description/urdf/_d585.urdf.xacro

```diff
@@ -47,11 +47,11 @@ aluminum peripherial evaluation case.
     <xacro:property name="d585_cam_depth" value="0.0931"/>
 
     <!-- TODO Samer: This parameter is still missing, took it from D455 -->
-    <xacro:property name="d585_cam_mount_from_center_offset" value="0.0377"/> <!-- TODO Samer: check -->
+    <xacro:property name="d585_cam_mount_from_center_offset" value="0.042"/> <!-- TODO Samer: check -->
```

Comment:
TODO Samer: check?

Reply:
Same for other lines

Reply:
We decided to keep these lines as log as we don't have the accurate numbers.

Reply:
fixed